### ISSUE TITLE
Replicate registry image

### ISSUE THREAD

**Issue by dimitar-kostadinov** [distribution/distribution-library-image#169]
**What would you like to be added**:
Consider replicate registry image to a public image registry that supports IPv6 natively and doesn't have a strict download rate limit.

**Why is this needed**:
In this way, the image can be used directly in production environment without the need of additional replication of the image.

**Comment by tianon** [comment]
Docker Hub has supported IPv6 for a little while now :sweat_smile:

https://www.docker.com/blog/docker-hub-registry-ipv6-support-now-generally-available/

**Comment by ialidzhikov** [comment]
Yes, Docker Hub supports IPv6 now, but we cannot use Docker Hub due to its [rate limiting](https://docs.docker.com/docker-hub/download-rate-limit/) for anonymous users - 100 pulls per 6 hours per IP address.

**Comment by milosgajdos** [comment]
All Docker Official Images are available on ECR Public https://gallery.ecr.aws/docker/library/registry but ECR still doesnt seem to support IPv6. Maybe if more interested is expressed in https://github.com/aws/containers-roadmap/issues/2287 it could get prioritised
